### DISCUSSION TITLE
audios with delay

### DISCUSSION THREAD

**Discussion by bruno918** [WhiskeySockets/Baileys#286] [General Questions] [answered]
hello, I have a problem where when I send audios, they take about 1 minute to be sent, does anyone know anything about this?

**Comment by joseorlandofp** [comment]
Is there any delay being sent with the message?
By delay I mean presence updates, like: before actually sending the audio, is it sending a recording presence?

I use it here with a lot of concurrent sessions and it does not take more than a few ms depending on filesize.

**Reply by bruno918** [reply]
Honestly I don't know what to do, my command is the following:
const audio = (url) => {
    bot.sendMessage(
      remoteJid,
      {
        audio: {
          url: url,
        },
        mimetype: "audio/mpeg",
        fileName: `EvoBOT`,
      },
      {
        quoted: baileysMessage,
      }
    );
  };

axios
      .get(`apilink}`)
      .then((res) => {
        sendWaitReply();
        image(
          res.data.result.thumb,
          `🧾 *Título:* ${res.data.result.title} \n 📺 *Canal:* ${res.data.result.channel} \n 👀 *Views:* ${res.data.result.views}`
        );
        let audiom = res.data.result.url;
        audio(audiom);
      })


The image sends instantly, I tested it with a 1 second audio and it sent instantly too, honestly, I have no more ideas.

**Comment by tb-sumberjaya** [comment, marked as answer]
I think it's caused by your audio source, try using another audio source (example: https://file-examples.com/storage/fee3d1095964bab199aee29/2017/11/file_example_MP3_5MG.mp3 ) to make sure.

**Reply by bruno918** [reply]
It was really this problem, I realized it now, thank you.
